Dalamud/Plugin/Internal/Profiles/PluginManagementCommandHandler.cs

```diff
@@ -236,10 +236,11 @@ void Continuation(Task t, string onSuccess, string onError)
                     .ConfigureAwait(false);
                 break;
             case PluginCommandOperation.Toggle:
-                this.chat.Print(Loc.Localize("PluginCommandsToggling", "Toggling plugin \"{0}\"...").Format(plugin.Name));
+                var isEnabling = plugin.State == PluginState.Unloaded;
+                this.chat.Print(Loc.Localize(isEnabling ? "PluginCommandsEnabling" : "PluginCommandsDisabling", "Toggling plugin \"{0}\"...").Format(plugin.Name));
```

Comment:
For EN, this'll still print `Toggling plugin {0}`, as there's no override. You'll need to set both the key (as you did) _and_ the default text for this case. As such, you might even want to split this out into multiple lines controlled by an `if` for readability rather than doing multiple ternaries.